tests/integration/pure_cairo_vm/test_runner_test/cheatable_syscall_handler_test.cairo

```diff
@@ -0,0 +1,19 @@
+func test_syscall_deployment_abi_storing() {
```

Comment:
The test's name doesn't tell why we need to store abi. I think this should be a test of `store` or/and `load` hint local.

Suggestion: `test_loading_from_contract_deployed_by_syscall`